Dockerfile

```diff
@@ -1,19 +1,20 @@
-FROM ubuntu:16.04
+FROM nfcore/base
 
 LABEL authors="rickard.hammaren@scilifelab.se, phil.ewels@scilifelab.se, martin.proks@scilifelab.se" \
     description="Docker image containing all requirements for NGI-RNAfusion pipeline"
 
 #Fusioncatcher 
 RUN apt-get -y update
 RUN apt-get -y install \
-    build-essential\
+    build-essential \
     libncurses5-dev \
     default-jdk \
     default-jre \
     gawk \
     gcc \
     g++ \
     bzip2 \
+    make \
```

Comment:
Is it normal to have `make` twice?
Personnaly, I sort the tools to install, that way, it's easier to spot duplicates

Reply:
Ahh, nice catch! 💪